omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/VisitResource1_9.java

```diff
@@ -309,10 +309,11 @@ public SimpleObject search(RequestContext context) throws ResponseException {
 	
 	private SimpleObject getVisits(RequestContext context, String patientParameter, String includeInactiveParameter,
 	        Date minStartDate) {
+		boolean includeAll = Boolean.parseBoolean(context.getParameter("includeAll"));
```

Comment:
@andela-cuchendu did you just forget that whatever you did in this commit? https://github.com/openmrs/openmrs-module-webservices.rest/commit/49ed4dc68b5ecef3dfd9cba4b88cfccfeeeb3bd3